js/coinbase.js

```diff
@@ -670,6 +670,39 @@ module.exports = class coinbase extends Exchange {
         return result;
     }
 
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const timestamp = this.seconds ();
+        const request = {
+            'currency': this.safeValue (params, 'currency', 'USD'),
+        };
+        const response = await this.publicGetExchangeRates (this.extend (request, params));
+        const data = this.safeValue (response, 'data', {});
+        const rates = this.safeValue (data, 'rates', {});
+        const result = {};
+        const coinKeys = Object.keys (rates);
+        for (let i = 0; i < coinKeys.length; i++) {
+            const coinName = coinKeys[i];
+            const value = rates[coinName]; // already in string, no need for safeNumber
```

Comment:
This line is a bit wrong – we always need safeNumber or safeString, cause we will switch from float-representations to string-representations everywhere eventually.